glass/observations.py

```diff
@@ -258,7 +251,7 @@ def equal_dens_zbins(
     cuml_nz /= cuml_nz[[-1]]
     zbinedges = np.interp(np.linspace(0, 1, nbins + 1), cuml_nz, z)
 
-    return list(zip(zbinedges, zbinedges[1:]))
+    return list(itertools.pairwise(zbinedges))
```

Comment:
TIL

Reply:
You can thank `ruff` https://docs.astral.sh/ruff/rules/zip-instead-of-pairwise